ui/analyse/src/study/topics.ts

```diff
@@ -80,6 +80,10 @@ export const formView = (ctrl: TopicsCtrl, userId?: string): VNode =>
         ],
       ),
     ],
+    onInsert: dlg => {
+      dlg.show();
+      (dlg.view.querySelector('.tagify__input') as HTMLElement)?.focus();
+    },
```

Comment:
I see that the fix works, so I'm merging it.

But I don't understand why it works. What does it have to do with `onInsert`? And the popup is still a `dialog`. Am confused

Reply:
Tagify doesn't work with HTMLDialogElement in modal mode, I think because it makes dodgy assumptions about stacking contexts and the power of z-index. I didn't look into if there was a flag or some way to customize how tagify inserts the dropdown into the DOM when just making the dialog non-modal did the trick.

It has nothing to do with onInsert except that's the [only way to show a non-modal snabDialog](https://github.com/lichess-org/lila/blob/fe905a15a405579f1e987fa482f2d250e14406a7/ui/common/src/dialog.ts#L44). See the comment about onInsert preventing showModal. Kind of a poor design decision.